breeze

```diff
@@ -2449,10 +2449,14 @@ function breeze::check_and_save_all_params() {
     if [[ -n "${INSTALL_AIRFLOW_REFERENCE=}" ]]; then
         if [[ ${INSTALL_AIRFLOW_REFERENCE=} == *1_10* ]]; then
             export BRANCH_NAME="v1-10-test"
+        elif [[ ${INSTALL_AIRFLOW_REFERENCE=} == *2_0* ]]; then
```

Comment:
We should continue using master until we continue rebasing. The "DEFAULT_BRANCH" in both will be master os overriding it here with v2-0-test will have side effects. for example few lines below we are limiting python version  to 3.6+ only when the DEFAULT_BRANCH is "master"